src/openms/include/OpenMS/FORMAT/DATAACCESS/SiriusMzTabWriter.h

```diff
@@ -79,7 +80,7 @@ namespace OpenMS
     //Output of Sirius is one directory per spectrum/compound
     //paths: Path to output directories of sirius
     //number: Amount of entries for each file/compound should be written to the mztab file
-    static void read(const std::vector<String> & paths, Size number, MzTab & result);
+    static void read(const std::vector<String> & paths, const String & mzml,  Size number, MzTab & result);
```

Comment:
documentation needs to be updated. there is no number parameter. others are not documented

Reply:
what does mzml mean in this context